test/files/presentation/doc/doc.check

```diff
@@ -0,0 +1,58 @@
+warning: 1 deprecation (since 2.13.0); re-run with -deprecation for details
+java.lang.AssertionError: assertion failed: doc/src is not a directory
```

Comment:
this stack trace in the check file seems accidental, or at least very prone to changes in the compiler that require updating the check file often